spk/python310/src/requirements.txt

```diff
@@ -0,0 +1,52 @@
+# Basic wheels
+setuptools==58.3.0
+pip==21.3.1
+wheel==0.37.0
+
+# cryptography dependencies
+idna==3.3
+ipaddress==1.0.23
+
+# Pure-python via spksrc.wheel.mk
+# commonly used wheels
+appdirs==1.4.4
+attrs==21.2.0
+beautifulsoup4==4.10.0
+certifi==2021.10.8
+chardet==4.0.0
+click==8.0.3
+distlib==0.3.3
+filelock==3.3.2
+Flask==2.0.2
+future==0.18.2
+html5lib==1.1
+idna==3.3
+importlib-metadata==4.8.1
+importlib-resources==5.3.0
+ipaddress==1.0.23
+itsdangerous==2.0.1
+Jinja2==3.0.2
+olefile==0.46
+oauthlib==3.1.1
+packaging==21.0
+paramiko==2.8.0
+platformdirs==2.4.0
+pyasn1==0.4.8
+pycparser==2.20
+PyJWT==2.3.0
+pyOpenSSL==21.0.0
+pyparsing==3.0.1
+python-dateutil==2.8.2
+pytz==2021.3
+requests==2.26.0
```

Comment:
We should pay attention that any packages moving to python310 remove these now included packages.
Thanks for including them :)

Reply:
fully agree with you.
there where so many used everywhere that I tried to capture the mostly used ones.